src/desktop/apps/article/components/InfiniteScrollArticle.js

```diff
@@ -135,7 +136,7 @@ export default class InfiniteScrollArticle extends React.Component {
   }
 
   renderContent = () => {
-    const { showTooltips, showToolTipMarketData } = this.props
+    const { showTooltips, showToolTipMarketData, onOpenAuthModal } = this.props
 
```

Comment:
so does this mean we will be passing `onOpenAuthModal` down the chain to individual tooltips?

Reply:
@eessex Yes, exactly. You can see how that's wired up in this PR https://github.com/artsy/reaction/pull/688